axes/utils.py

```diff
@@ -1,17 +1,32 @@
+import logging
+
 from platform import python_version
 from sys import platform
+
 if python_version() < '3.4' and platform == 'win32':
     import win_inet_pton
 from socket import inet_pton, AF_INET6, error
 
-from django.core.cache import cache
+from django.core.cache import cache, caches, InvalidCacheBackendError
```

Comment:
InvalidCacheBackendError is not used anywhere.

Reply:
Yes it is a remnant from an error I wanted to catch. I will remove it.